otherPages/categories/categories.css

```diff
@@ -1,131 +1,103 @@
 @font-face {
-    font-family: Righteous-Regular;
-    src: url("../../assets/font/Righteous-Regular.ttf");
-  }
-  * {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-  }
-  
-  a {
-      text-decoration: none;
-  }
-  
-  .header {
+  font-family: Righteous-Regular;
+  src: url("../../assets/font/Righteous-Regular.ttf");
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+a {
+    text-decoration: none;
+}
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2rem 1.875rem;
+}
+.header i {
+  font-size: 2.375rem;
+  color: #f79259;
+}
+.header .logo {
+  width: 4.25rem;
+  height: 3.8rem;
+}
+.title {
+  font-family: Righteous-Regular;
```

Comment:
I like the font family here but if you want to keep it, you should change the font family for other headers on other pages

Reply:
yes, In all Pages using this font